### PR TITLE
duffel bags of holding

### PR DESCRIPTION
**Pull request by Seris02**
## About The Pull Request

title

## Why It's Good For The Game

DUFFEL BAGS OF HOLDING GOOD YAS

## Changelog
:cl:
add: duffel bags of holding
/:cl:

### CODE REVIEW

**Comment by silicons** [comment]
as much as i like variety i do not want the boh storage issues to be worse.

File: code/modules/research/designs/bluespace_designs.dm [hunk 1]
```diff
@@ -35,6 +35,17 @@
 	dangerous_construction = TRUE
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
 
+/datum/design/duffelbag_holding
+	name = "Duffel Bag of Holding"
+	desc = "A duffel bag that opens into a localized pocket of bluespace."
+	id = "duffelbag_holding"
+	build_type = PROTOLATHE
+	materials = list(/datum/material/gold = 3000, /datum/material/diamond = 1500, /datum/material/uranium = 250, /datum/material/bluespace = 2000)
```
Comment (Trilbyspaceclone) [inline code/modules/research/designs/bluespace_designs.dm:43]: Holds more but costs the same?

Comment (Seris02) [reply code/modules/research/designs/bluespace_designs.dm:43]: eh, fine

**Comment by Ludox235** [comment]
Does it slow you down like normal duffel bags?

**Comment by silicons** [comment]
sorry regardless of slowdown we really do not need more boh storage creep.

**Review by silicons — APPROVED** [review]

**Pull request merged**